src/player/player.gd

```diff
@@ -135,21 +148,30 @@ func apply_move(delta):
 		motion.x = 0
 		
 
+# warning-ignore:unused_argument
 func apply_idle(delta):
+	get_node("StandingCollisionShape").disabled = false
```

Comment:
would it be better to do all the `get_node` calls in the `enter_state` function rather than every frame?

Reply:
Well the logic is that if man is not crouching, the collision shape should be standing. I could make it into 1 if statement.

Reply:
yeah but can't you just set it to false upon entering the state (in the _enter_state method)? otherwise it will get called every frame which is less efficient